strings/strip.py

```diff
@@ -0,0 +1,33 @@
+def strip(user_string: str, characters: str | None = None) -> str:
+    """
+    Remove leading and trailing characters (whitespace by default) from a string.
+
+    Args:
+        user_string (str): The input string to be stripped.
+        characters (str, optional): Optional characters to be removed
+                (default is whitespace).
+
+    Returns:
+        str: The stripped string.
+
+    Examples:
+        >>> strip("   hello   ")
+        'hello'
+        >>> strip("...world...", ".")
+        'world'
+        >>> strip("123hello123", "123")
+        'hello'
```

Comment:
```suggestion
        >>> strip("123hello123", "123")
        'hello'
        >>> strip("")
        ''
```
Adding a doctest for the empty string as an edge case

Reply:
Added the edge case.